### PR TITLE
correctly handled the no style case in streamProperties of OTSubscriber

### PR DESCRIPTION
**Pull request by mbouxin**
### Contributing checklist
- [x] Code must follow existing styling conventions
- [x] Added a descriptive commit message

### Solves issue(s)
#128

### CODE REVIEW

**Comment by kuy** [comment]
I'm waiting this PR is merged. Is there something problem? @msach22

**Comment by msach22** [comment]
@kuy Apologies for the delay on this, I'm aiming to release this early next week!

File: src/OTSubscriber.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ export default class OTSubscriber extends Component {
   render() {
     const childrenWithStreams = this.state.streams.map((streamId) => {
       const streamProperties = this.props.streamProperties[streamId];
-      const style = isEmpty(streamProperties) ? this.props.style : isNull(streamProperties.style) ? this.props.style : streamProperties.style;
+      const style = isEmpty(streamProperties) ? this.props.style : isUndefined(streamProperties.style) ? this.props.style : streamProperties.style;
```
Comment (msach22) [inline src/OTSubscriber.js]: @mbouxin shouldn't the change be `isNull() || isUndefined()`?

Comment (mbouxin) [reply src/OTSubscriber.js]: Indeed it would be more robust

**Pull request merged**